src/api/controllers/groupController.ts

```diff
@@ -14,38 +16,84 @@ export class GroupController extends BaseController {
         this.groupService = groupServiceInjected;
     }
 
-    async createHandler(req: RequestWithGroup, res: Response, next: NextFunction) {
+    async createGroup(req: RequestWithGroup, res: Response, next: NextFunction) {
         const group: BaseGroup = req.body;
         try {
             const createdGroup = await this.groupService.create(group);
             this.success(res, createdGroup);
         } catch (e) {
+            Logger.logControllerError('error', 'createGroup', 'Unable to create group',
```

Comment:
why did you create so complex interface for logging? I would recommend to define in `BaseController` property like `log` and initialize it with simple interface like `ILogger`

```
interface ILogger {
   error: () => {},
   info: () => {},
   warn: () => {},
   debug: () => {}
}
```

so, in the end the error declaration logic will look like the following:

```
...
} catch (e) {
    this.logger.error(`Group creation error -> "${e.getMessage()}"`)
}
```